deployment/hasura/metadata/databases/tables/actions/action_run.yaml

```diff
@@ -0,0 +1,47 @@
+table:
+  name: action_run
+  schema: actions
+configuration:
+  custom_name: "action_run"
+object_relationships:
+  - name: action_definition
+    using:
+      foreign_key_constraint_on: action_definition_id
+select_permissions:
+  - role: aerie_admin
+    permission:
+      columns: "*"
+      filter: {}
+      allow_aggregations: true
+  - role: user
+    permission:
+      columns: "*"
+      filter: {}
+      allow_aggregations: true
+  - role: viewer
+    permission:
+      columns: "*"
+      filter: {}
+      allow_aggregations: true
+insert_permissions:
+  - role: aerie_admin
+    permission:
+      columns: "*"
+      check: {}
+      set:
+        created_by: "x-hasura-user-id"
+  - role: user
+    permission:
+      columns: "*"
+      check: {}
+      set:
+        created_by: "x-hasura-user-id"
+update_permissions:
+  - role: aerie_admin
+    permission:
+      columns: [logs, error, results, status]
+      filter: {}
```

Comment:
Why are these columns editable? This comment is directed more towards the first three columns, as I can understand the admin updating `status` should, ie, the system go down while running an action.